ci/docker/runtime_functions.sh

```diff
@@ -413,6 +414,7 @@ build_ubuntu_cpu_mkl() {
         -DUSE_TVM_OP=ON \
         -DUSE_MKL_IF_AVAILABLE=ON \
         -DUSE_BLAS=MKL \
+        -DUSE_INT64_TENSOR_SIZE=OFF \
```

Comment:
Currently CPU:MKL build fails because LAPACK supports 32-bit architecture.
Cannot use Large Tensor with LAPACK and MKL. 
```
n file included from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/../linalg.h:31:0,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/la_op-inl.h:29,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/np_norm-inl.h:35,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/np_norm.cc:27:

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h: In function 'int MXNET_LAPACK_sgetri(int, int, float*, int, int*, float*, int)':

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:382:68: error: cannot convert 'int*' to 'const long long int*' for argument '5' to 'long long int LAPACKE_sgetri(int, long long int, float*, long long int, const long long int*)'

[2020-07-02T12:43:35.720Z]        return LAPACKE_##prefix##getri(matrix_layout, n, a, lda, ipiv); \

[2020-07-02T12:43:35.720Z]                                                                     ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:387:3: note: in expansion of macro 'MXNET_LAPACK_CWRAP_GETRI'

[2020-07-02T12:43:35.720Z]    MXNET_LAPACK_CWRAP_GETRI(s, float)

[2020-07-02T12:43:35.720Z]    ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h: In function 'int MXNET_LAPACK_dgetri(int, int, double*, int, int*, double*, int)':

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:382:68: error: cannot convert 'int*' to 'const long long int*' for argument '5' to 'long long int LAPACKE_dgetri(int, long long int, double*, long long int, const long long int*)'

[2020-07-02T12:43:35.720Z]        return LAPACKE_##prefix##getri(matrix_layout, n, a, lda, ipiv); \

[2020-07-02T12:43:35.720Z]                                                                     ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:388:3: note: in expansion of macro 'MXNET_LAPACK_CWRAP_GETRI'

[2020-07-02T12:43:35.720Z]    MXNET_LAPACK_CWRAP_GETRI(d, double)

[2020-07-02T12:43:35.720Z]    ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h: In function 'int MXNET_LAPACK_sgelsd(int, int, int, int, float*, int, float*, int, float*, float, int*, float*, int, int*)':

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:413:52: error: cannot convert 'int*' to 'long long int*' for argument '11' to 'long long int LAPACKE_sgelsd(int, long long int, long long int, long long int, float*, long long int, float*, long long int, float*, float, long long int*)'

[2020-07-02T12:43:35.720Z]                                       s, rcond, rank); \

[2020-07-02T12:43:35.720Z]                                                     ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:419:3: note: in expansion of macro 'MXNET_LAPACK_CWRAP_GELSD'

[2020-07-02T12:43:35.720Z]    MXNET_LAPACK_CWRAP_GELSD(s, float)

[2020-07-02T12:43:35.720Z]    ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h: In function 'int MXNET_LAPACK_dgelsd(int, int, int, int, double*, int, double*, int, double*, double, int*, double*, int, int*)':

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:413:52: error: cannot convert 'int*' to 'long long int*' for argument '11' to 'long long int LAPACKE_dgelsd(int, long long int, long long int, long long int, double*, long long int, double*, long long int, double*, double, long long int*)'

[2020-07-02T12:43:35.720Z]                                       s, rcond, rank); \

[2020-07-02T12:43:35.720Z]                                                     ^

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/.././c_lapack_api.h:420:3: note: in expansion of macro 'MXNET_LAPACK_CWRAP_GELSD'

[2020-07-02T12:43:35.720Z]    MXNET_LAPACK_CWRAP_GELSD(d, double)

[2020-07-02T12:43:35.720Z]    ^

[2020-07-02T12:43:35.720Z] In file included from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/../linalg.h:283:0,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/la_op-inl.h:29,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/np_norm-inl.h:35,

[2020-07-02T12:43:35.720Z]                  from /work/mxnet/src/api/operator/numpy/linalg/np_norm.cc:27:

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/../linalg_impl.h: In function 'void linalg_getrf(const mshadow::Tensor<Device, 2, DType>&, const mshadow::Tensor<xpu, 1, int>&, bool, mshadow::Stream<Device>*) [with xpu = mshadow::cpu; DType = float]':

[2020-07-02T12:43:35.720Z] /work/mxnet/src/api/operator/numpy/linalg/../../../../operator/numpy/linalg/../../tensor/../linalg_impl.h:1418:63: error: cannot convert 'int* const' to 'long long int*' for argument '6' to 'long long int LAPACKE_sgetrf(int, long long int, long long int, float*, long long int, long long int*)'

[2020-07-02T12:43:35.720Z]                                 A.dptr_, A.stride_, pivot.dptr_)); \

```

Reply:
Are we not using the MKL Lapack when compiling with MKL?

Reply:
@leezu : Lapack signatures are int32 only.

Reply:
That's not true. The signatures depend on the programming model. You need to specify ILP64 model.

Reply:
For OpenBLAS, the respective compile flag is `INTERFACE64=1`